packages/contracts/contracts/L1/SecurityCouncil.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { MultiSigWallet } from "../universal/MultiSigWallet.sol";
+import { Semver } from "../universal/Semver.sol";
+
+/**
+ * @custom:proxied
+ * @title SecurityCouncil
+ * @notice SecurityCouncil receives validation requests for specific output data,
+ *         and allows security council parties to validate & agree on transactions before execution.
+ */
+contract SecurityCouncil is MultiSigWallet, Semver {
+    /**
+     * @notice A mapping of output info array which have been submitted to validate.
+     */
+    mapping (uint => OutputInfo) public outputInfos;
+
+    /**
+     * @notice Emitted when an validation request is submitted.
+     *
+     * @param transactionId Index of the submitted transaction.
+     * @param outputRoot The output root.
+     * @param l2BlockNumber The L2 block number of the output root.
+     */
+    event Validation(
```

Comment:
```suggestion
    event ValidationRequested(
```

Reply:
I don't like this name either, but we did this to unify the naming with other existing events.